### PR TITLE
Download product

### PR DESCRIPTION
**Pull request by koenmetsu**

### CODE REVIEW

File: src/RoadRegistry.BackOffice.UI/src/DownloadProduct.elm [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+type alias Model =
+    { header : HeaderModel
+    , download : DownloadModel
+    , alert : AlertModel
+    }
+
+
+init : String -> ( Model, Cmd Msg )
+init url =
+    ( { header = Header.init |> Header.downloadBecameActive
```
Comment (yreynhout) [inline src/RoadRegistry.BackOffice.UI/src/DownloadProduct.elm:39]: There are now 2 download pages - think `Header.downloadBecameActive` will need to be deduped.

File: src/RoadRegistry.BackOffice.UI/src/DownloadExtract.elm [hunk 2]
```diff
@@ -36,7 +36,7 @@ type alias Model =
 
 init : String -> ( Model, Cmd Msg )
 init url =
-    ( { header = Header.init |> Header.downloadBecameActive
+    ( { header = Header.init |> Header.downloadExtractBecameActive
```
Comment (yreynhout) [inline src/RoadRegistry.BackOffice.UI/src/DownloadExtract.elm:39]: I'd prefer if `extract` was replaced with `editor` for now, as a term. Makes it consistent across the board even if not the best of names.

Comment (koenmetsu) [reply src/RoadRegistry.BackOffice.UI/src/DownloadExtract.elm:39]: `downloadForEditor` in that case?

Comment (koenmetsu) [reply src/RoadRegistry.BackOffice.UI/src/DownloadExtract.elm:39]: or just `downloadEditor`?

Comment (yreynhout) [reply src/RoadRegistry.BackOffice.UI/src/DownloadExtract.elm:39]: `downloadForEditor` has a nice ring to it.

**Comment by koenmetsu** [comment]
@yreynhout updated with rename to DownloadForEditor.

File: src/RoadRegistry.BackOffice.UI/src/Header.elm [hunk 3]
```diff
@@ -47,9 +48,14 @@ activateTitle title tabAction =
...
 
-downloadBecameActive : HeaderModel -> HeaderModel
-downloadBecameActive model =
-    { model | tabActions = List.map (activateTitle "Downloaden") model.tabActions }
+downloadForEditorBecameActive : HeaderModel -> HeaderModel
+downloadForEditorBecameActive model =
+    { model | tabActions = List.map (activateTitle "Download extract") model.tabActions }
+
+
+downloadProductBecameActive : HeaderModel -> HeaderModel
```
Comment (yreynhout) [inline src/RoadRegistry.BackOffice.UI/src/Header.elm:56]: `downloadForProductBecameActive` to keep the symmetry going?

**Pull request merged**